examples/mongodbatlas_federated_database_instance/main.tf

```diff
@@ -5,12 +5,20 @@ resource "aws_iam_role_policy" "test_policy" {
   {
     "Version": "2012-10-17",
     "Statement": [
-      {
-        "Effect": "Allow",
-		"Action": "*",
-		"Resource": "*"
-      }
-    ]
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:GetObject",
```

Comment:
Is the `s3:GetObject` and `s3:GetObjectVersion` necessary for all buckets?

Reply:
I found this documented somewhere, so yes. (even though I honestly haven't tried remove it)